Locker/Locker/Locker.h

```diff
@@ -12,29 +12,112 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface Locker : NSObject
 
+
+#pragma mark - Properties
+
+/**
+ Boolean value that indicates if biometric settings have changed
+ */
 @property (nonatomic, assign, class, readonly) BOOL biometricsSettingsAreChanged;
+
+/**
+ The biometrics type that the device supports (None, TouchID, FaceID).
+ */
 @property (nonatomic, assign, class, readonly) BiometricsType deviceSupportsAuthenticationWithBiometrics;
+
+/**
+ The biometrics type that the device supports which is enabled in the device settings.
+ */
```

Comment:
I'm not sure if this could be just a BOOL value? 
In conbination with `deviceSupportsAuthenticationWithBiometrics` you can check if user can use for example FaceID 🤔 
What do you think?